### ISSUE TITLE
Not include Plot for MS Word output if source file is in a Subdirectory

### ISSUE THREAD

**Issue by Lightbridge-KS** [quarto-dev/quarto-cli#1130]
### Bug description

Hi, I would like to give a bug report

- I use Quarto book to render a source file (`.qmd`) that lives in a **subdirectory**.
- I've found that, **plot output** from that particular file is not included if the output format is `docx`. (but HTML output works fine)

## Example

I've created [a minimal example repo here](https://github.com/Lightbridge-KS/quarto-book-figpath-bug).

`plot(cars)` from [this source file](https://github.com/Lightbridge-KS/quarto-book-figpath-bug/blob/main/subdir/intro.qmd) is not included in [MS Word output](https://github.com/Lightbridge-KS/quarto-book-figpath-bug/blob/main/_book/Quarto-debug-figure-path.docx) (but, [HTML output](https://lightbridge-ks.github.io/quarto-book-figpath-bug/) works fine)

The following error msg is generated:

```
[WARNING] Could not fetch resource intro_files/figure-docx/fig-cars-bug-1.png: replacing image with description
```

[**_quarto.yml**](https://github.com/Lightbridge-KS/quarto-book-figpath-bug/blob/main/_quarto.yml)

```yml
project:
  type: book

book:
  title: "Quarto debug figure path"
  author: "Kittipos Sirivongrungson"
  date: "6/11/2022"
  chapters:
    - index.qmd
    - subdir/intro.qmd # Figure problem in here

bibliography: references.bib

format:
  html:
    theme: cosmo
  docx: default
```

## My Session Info

```shell
$ quarto check
```

[✓] Checking Quarto installation......OK
      Version: 0.9.566
      Path: /Applications/quarto/bin

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.9.7 (Conda)
      Path: /Users/kittipos/opt/miniconda3/bin/python
      Jupyter: 4.9.2
      Kernels: julia-1.7, julia-1.5, python3

[✓] Checking Jupyter engine render....OK

[✓] Checking R installation...........OK
      Version: 4.2.0
      Path: /Library/Frameworks/R.framework/Resources
      LibPaths:
        - /Users/kittipos/Desktop/R_Programming/quarto/quarto-book-figpath-bug/renv/library/R-4.2/x86_64-apple-darwin17.0
        - /Library/Frameworks/R.framework/Versions/4.2/Resources/library
      rmarkdown: 2.14

[✓] Checking Knitr engine render......OK

```r
> session_info()
```
─ Session info ────────────────────────────────────
 setting  value
 version  R version 4.2.0 (2022-04-22)
 os       macOS Monterey 12.4
 system   x86_64, darwin17.0
 ui       RStudio
 language (EN)
 collate  en_US.UTF-8
 ctype    en_US.UTF-8
 tz       Asia/Bangkok
 date     2022-06-11
 rstudio  2022.02.3+492 Prairie Trillium (desktop)
 pandoc   2.16.2 @ /usr/local/bin/pandoc

Your guys did a great job so far.

Thank you very much.

### Checklist

- [X] [formatted your issue](https://yihui.org/issue/#please-format-your-issue-correctly) so it is easier for us to read?
- [X] included a minimal, self-contained, and reproducible example?
- [X] documented the quarto version you're running, by providing the output produced by `quarto check` in a terminal in your issue?
- [X] documented the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] documented which operating system you're running? If on Linux, please provide the specific distribution as well.
- [X] upgraded to the latest version, including your versions of R, the RStudio IDE, and relevant R packages?

**Comment by jjallaire** [comment]
Thanks for reporting this! I believe this results from the same underlying pandoc bug as this issue: https://github.com/quarto-dev/quarto-cli/issues/1019

The bug in pandoc is fixed however not yet available in a pandoc release so we may need to create a short-term workaround for our v1.0 release.

**Comment by Lightbridge-KS** [comment]
Thank you very much!

**Comment by ma-riviere** [comment]
Confirming that I have the same issue when knitting to docx

```
 version  R version 4.2.0 (2022-04-22 ucrt)
 os       Windows 10 x64 (build 22000)
 system   x86_64, mingw32
 ui       RStudio
 language (EN)
 collate  English_United States.utf8
 ctype    English_United States.utf8
 tz       Europe/Paris
 date     2022-06-12
 rstudio  2022.06.0-daily+405 Spotted Wakerobin (desktop)
 pandoc   2.18 @ D:/Program Files/RStudio/bin/quarto/bin/tools/ (via rmarkdown)
 Quarto   0.9.575
```

**Comment by jjallaire** [comment]
Thanks again for reporting this, should be fixed here: https://github.com/quarto-dev/quarto-cli/commit/6e5f5652f844b7d80eb400af67e78ab24f4cbbe1 (fix available in build v0.9.576: https://github.com/quarto-dev/quarto-cli/releases/latest)

**Comment by Lightbridge-KS** [comment]
Problem is solved, thanks!
